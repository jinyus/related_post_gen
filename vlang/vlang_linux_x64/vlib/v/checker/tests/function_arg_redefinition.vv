fn f(para1 int, para1 f32) int {
	return para1
}

fn main() {
	a := f(11, 1.1)
}
