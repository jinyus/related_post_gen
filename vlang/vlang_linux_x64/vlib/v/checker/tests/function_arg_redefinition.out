vlib/v/checker/tests/function_arg_redefinition.vv:1:17: error: redefinition of parameter `para1`
    1 | fn f(para1 int, para1 f32) int {
      |                 ~~~~~
    2 |     return para1
    3 | }
